chrome-launcher.ts

```diff
@@ -337,7 +350,9 @@ export class Launcher {
         delete this.errFile;
       }
 
-      this.rimraf(this.userDataDir, () => resolve());
+      if (typeof this.userDataDir === 'string') {
```

Comment:
maybe just check `this.forceDefaultProfile`?

Reply:
Need to do that check or otherwise TypeScript will complain it can be `Boolean` or `string`

Reply:
well we can always keep userDataDir a string inside this class, it's just `boolean|string` in `this.opts`